package.json

```diff
@@ -95,10 +95,7 @@
       ]
     },
     "server-headful": {
-      "command": "./runBiDiServer.sh --headless=false",
-      "files": [
-        "runBiDiServer.sh"
-      ],
+      "command": "npm run server-headless -- --headless=false",
```

Comment:
If it works, `npm run server --headless=false` seems less odd.

Reply:
It doesn't work ;)